code/game/objects/items/shields.dm

```diff
@@ -256,62 +260,30 @@
 	shield_flags = SHIELD_FLAGS_DEFAULT
 	max_integrity = 300
 
+//Bulletproof riot shield
 obj/item/shield/riot/bullet_proof
 	name = "bullet resistant shield"
 	desc = "A far more frail shield made of resistant plastics and kevlar meant to block ballistics."
 	armor = list("linemelee" = 80, "linebullet" = 400, "laser" = 0, "energy" = 0, "bomb" = -40, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 50)
-	max_integrity = 300
-
-/obj/item/shield/riot/roman
-	name = "\improper Roman shield"
-	desc = "Bears an inscription on the inside: <i>\"Romanes venio domus\"</i>."
-	icon_state = "roman_shield"
-	item_state = "roman_shield"
-	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
-	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
-	repair_material = /obj/item/stack/sheet/mineral/wood
+	icon_state = "riot_bulletproof"
+	item_state = "riot_bulletproof"
 	shield_flags = SHIELD_FLAGS_DEFAULT
-	max_integrity = 250
-
-/obj/item/shield/riot/roman/fake
```

Comment:
I don't recommend removing the TG roman shields. They are used in some places on the map and will break things.